react/TextField/TextField.js

```diff
@@ -72,20 +74,22 @@ export default class TextField extends Component {
     this.handleMouseDownOnClear = this.handleMouseDownOnClear.bind(this);
   }
 
+  storeContainerReference = textField => {
+    if (textField !== null) {
+      this.container = textField;
+    }
+  }
+
   storeInputReference(input) {
     if (input !== null) {
       this.input = input;
     }
   }
 
   handleMouseDownOnClear(event) {
-    const { onClear } = this.props;
-
-    if (typeof onClear === 'function') {
```

Comment:
That's what I thought at first, but the logic has actually changed now. Previously, this whole block would be skipped if `onClear` wasn't defined, which isn't the case anymore. Is that desired behaviour?